### ISSUE TITLE
configurer le hostname definitif : https://authentification.beta.gouv.fr/

### ISSUE THREAD

**Issue by pseudonyme21** [tchapgouv/tchap-identite#56]

**Comment by jdauphant** [comment]
How did you discriminate between authentification.beta.gouv.fr , identification.beta.gouv.fr  or identite.beta.gouv.fr , does the design team get involved ?

**Comment by odelcroi** [comment]
@jdauphant Design team have designed mockups here with the hostname  https://authentification.beta.gouv.fr/, I guess we can go with it?

**Comment by jdauphant** [comment]
> @jdauphant Design team have designed mockups [here](https://www.figma.com/proto/3qu9dfXgLomL6u04mpaTip/Tchap---v4-(Tatin)?page-id=601%3A2165&node-id=1360%3A5940&viewport=-5305%2C-2928%2C0.31&scaling=min-zoom&starting-point-node-id=1360%3A6083&show-proto-sidebar=1) with the hostname https://authentification.beta.gouv.fr/, I guess we can go with it?
The mockup was made for usertesting on friday. We should double check with them for final version.

**Comment by Caroline-lawson** [comment]
Yes, participants spoke about authentification and identification. We chose authentification because it is more accurate.

**Comment by jdauphant** [comment]
@pseudonyme21 @odelcroi authentification.beta.gouv.fr and authentification.incubateur.net have been delegated to the OVH account
